docker/java/spring/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM maven:3.5.3-jdk-10
+
+RUN mkdir /agent \
+    && mkdir /app
+
+COPY testapp /app
+
+WORKDIR /agent
+
+RUN curl -L https://github.com/elastic/apm-agent-java/archive/master.tar.gz | tar --strip-components=1 --exclude=demo/db.sql.bz2 -xzv \
+    && mvn package -DskipTests \
```

Comment:
add `--batch-mode`
This prevents the download progress indicators which blow up the logs considerably

Reply:
Done